components/cta/TokenUpdated.vue

```diff
@@ -2,16 +2,14 @@
   <section>
   <div class="container-parent">
     <div class="container-child" flex row justify-center align-center>
-      <div class="content" v-for="exchange in exchanges" :key="exchange.exchanges" flex column justify-center align-center>
+      <div class="content" flex column justify-center align-center>
         <h2>
-          Token available on
+          Buy and trade the MESG Token
         </h2>
-        <img :src="exchange.svg" :alt="exchange.title" class="logo" mb2/>
-        <p mb2>
-          The MESG Token is now available on DigiFinex. Introducing the token for the new economy of development.
-        </p>
-        <nav flex>
-          <Button href="https://www.digifinex.com/en-ww/trade/BTC/MESG" target="_blank" class="btn" primary>Buy the MESG Token</Button>
+        <nav class="exchanges" flex row justify-center>
+          <Button href="https://www.digifinex.com/en-ww/trade/USDT/MESG" target="_blank" white flex column mr1 mt1><img src="~/assets/Digifinex.svg" alt="Digifinex" class="logo"/></Button>
```

Comment:
this will solve the compilation issue. Problem with capitalization of the file (this is not valid on linux environment)
```suggestion
          <Button href="https://www.digifinex.com/en-ww/trade/USDT/MESG" target="_blank" white flex column mr1 mt1><img src="~/assets/digifinex.svg" alt="Digifinex" class="logo"/></Button>
```